AddisAbabaNALE/NLA24.xml

```diff
@@ -64,68 +72,89 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
-                     <source>
-                        <listBibl type="catalogue"><bibl><ptr target="bm:Unesco1970Manuscripts"/></bibl><bibl><ptr target="bm:NALE1962"/><citedRange>13-14</citedRange></bibl></listBibl>
-                        <listBibl type="secondary"><bibl><ptr target="bm:ErhoStuckenbruck2013Enoch"/><citedRange>125</citedRange></bibl></listBibl>
-                        
-                     </source>
-                  </recordHist>
-               </adminInfo>
-            </additional>   
+               <additional>
+                  <adminInfo>
+                     <recordHist>
+                        <source>
+                           <listBibl type="catalogue">
+                              <bibl><ptr target="bm:Unesco1970Manuscripts"/></bibl>
+                              <bibl><ptr target="bm:NALE1962"/><citedRange>13-14</citedRange></bibl>
+                           </listBibl>
+                           <listBibl type="secondary">
+                              <bibl><ptr target="bm:ErhoStuckenbruck2013Enoch"
+                                    /><citedRange>125</citedRange></bibl>
+                           </listBibl>
+                        </source>
+                     </recordHist>
+                  </adminInfo>
+               </additional>
             </msDesc>
-            
+
          </sourceDesc>
       </fileDesc>
       <encodingDesc>
          <projectDesc>
             <p>Encoded according to TEI P5 Guidelines.</p>
          </projectDesc>
-         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude"
+            href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
             <xi:fallback>
                <p>Definitions of prefixes used.</p>
             </xi:fallback>
          </xi:include>
       </encodingDesc>
       <profileDesc>
-         <langUsage><language ident="en">English</language></langUsage>
-         <textClass><keywords><term key="ChristianLiterature"></term><term key="Bible"></term><term key="OldTestament"></term></keywords></textClass>
+         <langUsage>
+            <language ident="en">English</language>
+         </langUsage>
+         <textClass>
+            <keywords>
+               <term key="ChristianLiterature"/>
+               <term key="Bible"/>
+               <term key="OldTestament"/>
+            </keywords>
+         </textClass>
       </profileDesc>
       <revisionDesc>
 
-         <change when="2023-04-27" who="ES">created stub from EMIP spreadsheet and vHMML metadata by Ted Erho</change>
+         <change when="2023-04-27" who="ES">created stub from EMIP spreadsheet and vHMML metadata by
+            Ted Erho</change>
+         <change who="NV" when="2024-12-16">added ID LIT1340EnochE based on bm:ErhoStuckenbruck2013Enoch</change>
```

Comment:
There is no need to base on the bibliography mentioned as the full content is already provided in the summary 
EMML: Enoch, fol. 1r-55r; 1 Maccabees, fol. 55v-95v; 2 Maccabees, fol.
                     95v-122r; 3 Maccabees, fol. 122r-138v
and can be edited to msItems if needed - either the whole or none. 
add bm:vHMML to bibliography wherever applicable

Reply:
ok!